src/pages/solution/adobe-campaign-classic/tabs/push-payloads.md

```diff
@@ -0,0 +1,161 @@
+---
+noIndex: true
+---
+
+<Variant platform="fcm" template="basic" repeat="9"/>
+
+### Configuration
+
+If using "remind later" functionality in a basic notification, it is required that you declare use of the `USE_EXACT_ALARM` permission in your `AndroidManifest.xml`:
+
+```xml
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.mycompany.myapp">
+    ...
+    <uses-permission android:name="android.permission.USE_EXACT_ALARM" />
+    ...
+</manifest>
+```
+
+### Properties
+
+The properties below are used to define the payload sent to FCM:
+
+| **Field** | **Required** | **Key** | **Type** | **Description** |
+| :-------- | :----------- | :------ | :------- | :-------------- |
+| Payload Version | ✅ | `adb_version` | string | Version of the payload assigned by the ACC authoring UI. |
+| Template Type | ✅ | `adb_template_type` | string | Informs the reader which properties may exist in the template object.<br />Basic template uses a value of "basic". |
+| Title | ✅ | `adb_title` | string | Text shown in the notification's title. |
+| Color - Title | ⛔️ | `adb_clr_title` | string | Text color for `adb_title`.<br />Represented as six character hex, e.g. `00FF00` |
+| Body | ✅ | `adb_body` | string | Text shown in message body when notification is collapsed. |
+| Expanded Body | ⛔️ | `adb_body_ex` | string | Body of the message when the message is expanded. |
+| Color - Body | ⛔️ | `adb_clr_body` | string | Text color for `adb_body`, `adb_body_ex`.<br />Represented as six character hex, e.g. `00FF00` |
+| Sound | ⛔️ | `adb_sound` | string | Sound played when notification is delivered. |
```

Comment:
i think it would be good to add information regarding the filename and where the file should be placed. For Android, this value should be the filename **without** the extension and the file should be placed in the app's `assets/raw` directory.